index.js

```diff
@@ -68,7 +68,8 @@ class Jarvis {
       handler: handler,
       help: help,
       tokens: parseCommand(command),
-      patterns
+      patterns,
+      description
```

Comment:
shouldn't we store this value somewhere in the object?

Reply:
@hliyan  I didn't find a requirement to save this. I have test that whenever we take all the commands using ``` Jarvis.commands ```  description is there.